gulpfile.js

```diff
@@ -77,7 +78,7 @@ gulp.task('deploy-vsix', ['package', 'compute-hashes'], function() {
         password: ARTIFACTORY_DEPLOY_PASSWORD,
         properties: {
           'vcs.revision': BUILD_SOURCEVERSION,
-          'vcs.branch': BUILD_SOURCEBRANCH,
+          'vcs.branch': SYSTEM_PULLREQUEST_TARGETBRANCH || BUILD_SOURCEBRANCH,
```

Comment:
```suggestion
          'vcs.branch': SYSTEM_PULLREQUEST_SOURCEBRANCH || BUILD_SOURCEBRANCH,
```

Reply:
I disagree. If you look at the [parent pom](https://github.com/SonarSource/parent-oss/commit/ea6a1dc54f41ac922f458fcf802bb752b0af4c8e#diff-600376dffeb79835ede4a0b285078036R676), the variable was `TRAVIS_BRANCH` which for a PR is the target branch.